stable/chaoskube/values.yaml

```diff
@@ -20,6 +20,9 @@ interval: 10m
 # label selector to filter pods by, e.g. app=foo,stage!=prod
 labels:
 
+# kinds selector to filter pods by, e.g. '!DaemonSet' (use quotes)
```

Comment:
Are quotes really need in yaml as well?

Reply:
Hmm, I remember there was a reason for it. Trying to remember...

Reply:
I can see the same note for the `namespaces` filter.

Reply:
Alright, then let's keep it.